src/main/java/org/team1540/robot2024/subsystems/indexer/IndexerIO.java

```diff
@@ -0,0 +1,33 @@
+package org.team1540.robot2024.subsystems.indexer;
+
+import org.littletonrobotics.junction.AutoLog;
+
+public interface IndexerIO {
+
+    @AutoLog
+    class IndexerIOInputs {
+        public double intakeVoltage;
+        public double intakeCurrentAmps;
+        public double intakeVelocityRPM;
+        public double feederVoltage;
+        public double feederCurrentAmps;
+        public double feederVelocityRPM;
+        public boolean noteInIntake = false;
+        public double setpointRPM;
+        public double feederPositionError;
```

Comment:
The setpoint logging should happen in the subsystem, not the IO inputs. Also, why are you logging the position error? Isn't the feeder only velocity controlled?